src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -67,7 +67,7 @@ type Alert implements Node {
     stop: Stop
```

Comment:
These stop, route, agency and trip fields need to be replaced with stops etc because alerts can have multiple entities but I'm not sure what is the ideal way. Should the stops etc field be at the root level of the alert of under some entities field in some way.

Reply:
We discussed this in today's meeting. @hannesj suggested that we should implement the entities similarly as the places in the nearest query so you need to use fragments to handle the different entity types.

Reply:
Implemented in https://github.com/opentripplanner/OpenTripPlanner/pull/3747/commits/1ee1514434c5f3fce3351213ea827525f722914e